templates/web-application/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "{{PACKAGE-NAME}}",
+  "version": "0.0.0",
+  "description": "",
+  "type": "module",
+  "scripts": {
+    "dev": "webpack serve --config webpack.dev.js",
+    "build": "webpack --config webpack.build.js",
+    "test": "jest",
+    "serve-build": "pnpm http-server dist -p 8080 -P http://localhost:8080?",
+    "reset": "pnpm dlx rimraf node_modules dist",
+    "lint": "eslint .",
+    "analyze": "webpack --config webpack.build.js --analyze",
+    "postinstall": "pnpm dlx rimraf node_modules/.cache"
+  },
+  "dependencies": {
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-router-dom": "6.11.0"
+  },
+  "devDependencies": {
+    "@pmmmwh/react-refresh-webpack-plugin": "^0.5.10",
+    "@swc/core": "^1.3.56",
```

Comment:
I removed `@swc/cli``vs https://github.com/patricklafrance/pnpm-ts5-swc-esm-jest/blob/master/package.json because it's not used when using `WebPack`